examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js

```diff
@@ -74,7 +74,11 @@ export default class MeshLayer extends SimpleMeshLayer {
     }
 
     const {viewport} = this.context;
-    const {sizeScale, coordinateSystem, _instanced} = this.props;
+    const {sizeScale, coordinateSystem, _instanced, viewportIds} = this.props;
+
+    if (!viewportIds.includes(viewport.id)) {
```

Comment:
I am not sure why we need to do this inside the layer - can't we use the layerFilter?

Reply:
layerFilter only works for top-level layers. The MeshLayer instance is part of the composite TileLayer instance. So we only see TileLayer in the layer filter.